crates/solidity/testing/perf/benches/iai/dataset.rs

```diff
@@ -243,3 +246,55 @@ pub fn run_query(trees: &Vec<Node>) {
         "Function names don't match: {results:#?}"
     );
 }
+
+struct NoOpResolver;
+
+impl PathResolver for NoOpResolver {
+    fn resolve_path(&self, _context_path: &str, path_to_resolve: &str) -> Option<String> {
+        Some(path_to_resolve.to_string())
+    }
+}
+
+pub fn run_create_bindings() {
+    let _ = bindings::create_with_resolver(SOLC_VERSION, Arc::new(NoOpResolver {}));
+}
+
+pub fn run_bindings(trees: &Vec<Node>) -> Vec<Bindings> {
+    let mut result = vec![];
+    let mut definition_count = 0_usize;
+
+    for tree in trees {
+        let mut bindings = bindings::create_with_resolver(SOLC_VERSION, Arc::new(NoOpResolver {}));
+        bindings.add_file("input.sol", tree.cursor_with_offset(TextIndex::ZERO));
+        definition_count += bindings.all_definitions().count();
+        result.push(bindings);
+    }
+
+    // 723 definitions
+    println!("A total of {definition_count} definitions were found");
```

Comment:
Just thinking out loud:

I guess these counts will not stabilizie anytime soon, and they will only go up as we complete the bindings.
But just for safety/to make sure the tests actually run E2E, WDYT of at least asserting they are above a minimum number?
We can replace it with the final count/list once the bindings are complete.

```suggestion
    assert!(definition_count > THE_CURRENT_COUNT, "Only found {definition_count} definitions");
```